utils/filter.h

```diff
@@ -143,8 +145,11 @@ void uftrace_setup_caller_filter(char *filter_str, struct symtabs *symtabs,
 				 struct rb_root *root,
 				 struct uftrace_filter_setting *setting);
 void uftrace_setup_hide_filter(char *filter_str, struct symtabs *symtabs,
-				 struct rb_root *root,
-				 struct uftrace_filter_setting *setting);
+			       struct rb_root *root,
+			       struct uftrace_filter_setting *setting);
```

Comment:
Unnecessary changes.

Reply:
no need to align?

Reply:
That means it's not related to this change so can be fixed later in a separate patch.